ci-operator/step-registry/openshift-qe/orion/cudn-density/openshift-qe-orion-cudn-density-commands.sh

```
@@ -0,0 +1,345 @@
+
+pushd orion
+pip install -q --retries "$MAX_RETRIES" -r requirements.txt
+pip install -q --retries "$MAX_RETRIES" .
+popd
+
+pushd orion
+
+# Invoked from orion repo by the openshift-ci bot
+if [[ -n "${PULL_NUMBER-}" ]] && [[ "${REPO_NAME}" == "orion" ]]; then
+  echo "Invoked from orion repo by the openshift-ci bot, switching to PR#${PULL_NUMBER}"
+  git pull origin pull/${PULL_NUMBER}/head:${PULL_NUMBER} --rebase
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)